bundle-workflow/src/test_workflow/integ_test/local_test_cluster.py

```diff
@@ -92,7 +92,7 @@ def wait_for_service(self):
                 logging.info(f"Pinging {url} attempt {attempt}")
                 response = requests.get(url, verify=False, auth=("admin", "admin"))
                 logging.info(f"{response.status_code}: {response.text}")
-                if response.status_code == 200 and '"status":"green"' in response.text:
```

Comment:
Check if the cluster is yellow/green. 
We do not want to proceed if the cluster is red.

Reply:
I suppose the `status_code` will be 200 only when the cluster is green/yellow. Correct me if I am wrong here

Reply:
it can be 200 and red

Reply:
Correct status code will be 200, just verify `response.text` has green/yellow.

Reply:
Changed the message to `Service is available`